src/smoosh/src/smoosh_channel.erl

```diff
@@ -282,6 +285,10 @@ handle_info(unpause, State) ->
 terminate(_Reason, _State) ->
     ok.
 
+persist_queue(#state{waiting = Waiting} = State) ->
+    write_state_to_file(State),
```

Comment:
Should we move the log statement ass well?

Reply:
never mind you did it in 9f43aa4573ca88d4d

Reply:
Added two log statements:
1. https://github.com/apache/couchdb/pull/3766/commits/5c737f17e96d9a331fef2d6cfa8964b25ee8c53a#diff-113cd247cfe56b1aa882a37daa2856fd33bab2e2a6183034eb3a30088960dc13R342
2. https://github.com/apache/couchdb/pull/3766/commits/5c737f17e96d9a331fef2d6cfa8964b25ee8c53a#diff-e1a750216ea14302008b3dfa1bbdec43a968fcb06939d6b91b1404fd37273091R57